src/Microsoft.DotNet.MSBuildSdkResolver/Interop.NETStandard.cs

```diff
@@ -16,6 +17,16 @@ internal static partial class Interop
     {
         internal static readonly bool RunningOnWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
 
+        internal static string realpath (string path)
+        {
+            var buffer = new StringBuilder(4096); // PATH_MAX from linux/limits.h
```

Comment:
@nguerrera I decided to stick with the realpath() approach.

A quick test showed that just p/invoking into `realpath()` with null as the second argument would not leak the string so I guess the marshalling infrastructure takes care of freeing the native string? I'll admit that I'm not an expert in this area so I used the explicit free(). Let me know if I should change it.